### PR TITLE
loader: change the proto package to be the gunk package

### PR DESCRIPTION
**Pull request by vishen**
Change the proto file descriptor package to be the same as the Gunk
package. Previously the proto file descriptor package was the path of
the Gunk file, it will now be whatever is defined as the Gunk package.

### CODE REVIEW

File: loader/loader.go [hunk 1]
```diff
@@ -632,6 +633,19 @@ func (l *Loader) convertEnum(tspec *ast.TypeSpec) (*desc.EnumDescriptorProto, er
 	return enum, nil
 }
 
+// formatTypeForPackage will format the type name for that package. If the
+// package is nil, it will format the type for the current file / package that is
+// being processed.
+//
+// Currently we format the type as ".<pkg_name>.<type_name>"
+func (l *Loader) formatTypeForPackage(typeName string, pkg *types.Package) string {
```
Comment (mvdan) [inline loader/loader.go]: I think that a name like `qualifiedTypeName` would be more descriptive of what the purpose here is.

File: loader/loader.go [hunk 2]
```diff
@@ -632,6 +633,19 @@ func (l *Loader) convertEnum(tspec *ast.TypeSpec) (*desc.EnumDescriptorProto, er
 	return enum, nil
 }
 
+// formatTypeForPackage will format the type name for that package. If the
+// package is nil, it will format the type for the current file / package that is
```
Comment (mvdan) [inline loader/loader.go]: I'd drop the `file / package` bit in favor of just either option, as that makes the sentence unnecessarily complex to parse. We care about the package name, so I'd just say "current package".

File: loader/loader.go (hunk 1 above)
Comment (vishen) [reply loader/loader.go]: Done.

File: loader/loader.go (hunk 2 above)
Comment (vishen) [reply loader/loader.go]: Done.

**Pull request merged**